src/main/java/com/dumbdogdiner/stickyapi/bukkit/nms/PacketInjector.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2020-2021 DumbDogDiner <dumbdogdiner.com>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.bukkit.nms;
+
+import io.netty.channel.*;
+import org.bukkit.entity.Player;
+
+import java.lang.reflect.Field;
+
+/**
+ * Handles Packet Injection
+ */
+public class PacketInjector {
```

Comment:
`@NotNull` annotations where needed please uwu
See: [this](https://www.jetbrains.com/help/idea/nullable-and-notnull-annotations.html#notnull)